desktop/src/com/unciv/app/desktop/DesktopLauncher.kt

```diff
@@ -23,6 +23,8 @@ internal object DesktopLauncher {
         // Solves a rendering problem in specific GPUs and drivers.
         // For more info see https://github.com/yairm210/Unciv/pull/3202 and https://github.com/LWJGL/lwjgl/issues/119
         System.setProperty("org.lwjgl.opengl.Display.allowSoftwareOpenGL", "true")
+        // This setting (default 64) limits clipboard transfers. Value in kB!
```

Comment:
If this is an arbitrary number please write that so we know later we can change it
If not please look to relevant source

Reply:
Maybe even make it a setting to allow our support hotline to just say 'open your config in VIM and add...'... Nah. Maybe one day I have the Eureka on how to bypass Gdx calling the wrong lwjgl3 function.

Reply:
The lwjgl3 glue layer to GLFW has a glfwSetClipboardString(long, CharSequence) and a glfwSetClipboardString(long, ByteBuffer) overload, and only the former does that despicable MemoryStack.nUTF8 conversion.

Reply:
lwjgl3, not GLFW - the file is called that, but it's the lwjgl3 glue to the latter. Yes of course, I just meant Gdx would have a change to mitigate, but we no longer have. Not in a way I can think of. Well, short of of forking and patching. Or PR'ing a patch to them and waiting....................

Reply:
Yea but... That clipboard stuff gets complicated once you dive deep, and some of that abstraction I wouldn't want to replicate. So -> ask lwjgl3 directly, but if we pull in those dependencies, will that coexist nicely??? That's where my brain says "SEP".